.changeset/silent-scissors-suffer.md

```diff
@@ -0,0 +1,9 @@
+---
+'@primer/brand-primitives': minor
+'@primer/react-brand': minor
+---
+
+⚠️ This update contains a breaking change.
+
+- Added new default behaviour to the `variant` prop of the `LogoSuite` component. If `variant` is not provided then either `emphasis` or `muted` styles are automatically applied depending on the number of logos in the suite. 5 or fewer uses `emphasis`, 6 or more uses `muted`.
```

Comment:
Minor stylistic/grammatical change around using numerals, particularly at the start of a sentence.

```suggestion
- Added new default behaviour to the `variant` prop of the `LogoSuite` component. If `variant` is `undefined` then either `emphasis` or `muted` styles are automatically applied depending on the number of logos in the `LogoBar`. Five or fewer logos apply the `emphasis` style, while six or more use the `muted` style.

```

Reply:
Added, thanks!